conf/modules/nav_rotorcraft.xml

```diff
@@ -11,7 +11,7 @@
       <dl_settings NAME="NAV">
         <dl_setting var="flight_altitude" MIN="0" STEP="0.1" MAX="999" module="navigation" unit="m" handler="SetFlightAltitude"/>
         <dl_setting var="nav.heading" MIN="0" STEP="0.1" MAX="360" unit="rad" alt_unit="deg"/>
-        <dl_setting var="nav.radius" MIN="-50" STEP="0.1" MAX="50" unit="m"/>
+        <dl_setting var="nav.radius" MIN="50" STEP="1" MAX="1000" unit="m"/>
```

Comment:
Please don't change the parameters here. The sign of the radius tells the rotation direction, so here you can only turn clockwise with a minimum radius of 50 meters.
If you need this setting specifically for the rotwing, you can add it in a separated settings file or add it to one of the rotwing-specific modules. It is not a problem if the settings appear multiple times.